### PR TITLE
MF-1565 - Document Bearer, Thing and Basic Authorization header

### LINKED ISSUE #1565: Document Bearer Authorization header
**FEATURE REQUEST**

2. Describe the feature you are requesting, as well as the possible use case(s) for it.
Document Bearer Authentication header. This is related to #1523 documentation.

3. Indicate the importance of this feature to you (must-have, should-have, nice-to-have).
must have

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### What does this do?
Document mandatory Bearer authz header
### Which issue(s) does this PR fix/relate to?
Resolve #1565

### CODE REVIEW

**Review by drasko — CHANGES_REQUESTED** [review]
LGTM

File: api/openapi/bootstrap.yml [hunk 1]
```diff
@@ -533,3 +501,31 @@ components:
...
+        * Users access: "Authorization: Bearer <user_token>"
+
+    thingAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: uuid
+      description: |
+        * Things access: "Authorization: Thing <external_key>"
+
+    thingEncAuth:
```
Comment (dborovcanin) [inline api/openapi/bootstrap.yml]: Probably `bootstrapAuth` is a better naming.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: api/openapi/bootstrap.yml (hunk 1 above)
Comment (manuio) [reply api/openapi/bootstrap.yml:520]: Probably better yes

**Review by dborovcanin — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**